checks/check_status.py

```diff
@@ -127,7 +127,8 @@ def validate_api_key(config):
         if proxy:
             request_proxy = {'https': "http://{user}:{password}@{host}:{port}".format(**proxy)}
         r = requests.get("%s/api/v1/validate" % config['dd_url'].rstrip('/'),
-            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3)
+            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3,
+                         verify=(not config['skip_ssl_validation']))
```

Comment:
We have a small flake8 issue here: `E126 continuation line over-indented for hanging indent`